optimum/exporters/openvino/model_patcher.py

```diff
@@ -1493,15 +1493,54 @@ def _phi3_self_attn_sdpa_forward(
     return attn_output, None, past_key_value
 
 
+# @torch.jit.script
```

Comment:
I think it makes sense to add the test with long prompt. Is this issue reproduced on tiny-model?